### PR TITLE
Resolve commented typos and formatting issues.

### PR DESCRIPTION
**Pull request by VirajSalaka**
### Purpose
Fix Formatting issues and typos.
https://github.com/wso2/product-microgateway/pull/1132
https://github.com/wso2/product-microgateway/pull/1110
https://github.com/wso2/product-microgateway/pull/1035
https://github.com/wso2/product-microgateway/pull/1004
https://github.com/wso2/product-microgateway/pull/727
https://github.com/wso2/product-microgateway/pull/562
https://github.com/wso2/product-microgateway/pull/503
https://github.com/wso2/product-microgateway/pull/452
https://github.com/wso2/product-microgateway/pull/996

### Issues

Fixes #

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

**Comment by bhathiya** [comment]
@VirajSalaka why is this still open?

File: components/micro-gateway-cli/src/main/resources/templates/service.mustache [hunk 1]
```diff
@@ -316,7 +316,6 @@ service {{cut qualifiedServiceName " "}} on {{#api.transport}}{{#equals . "http"
                                 message : "Runtime Error",
                                 description : errorDescription
                             }};
-
                             res.setPayload(payload);
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/resources/templates/service.mustache:319]: better to keep the empty line to separate this line from payload creation and make it more readable.
```suggestion

                res.setPayload(payload);
```

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/utils/OpenAPICodegenUtils.java [hunk 2]
```diff
@@ -911,6 +911,13 @@ private static boolean validateAppSecurityOptionality(ApplicationSecurity appSec
         return new String[]{securitySchemas, scopes};
     }
 
+    /**
+     * Get resource level auth providers as a string array.
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/utils/OpenAPICodegenUtils.java:915]:
```suggestion
     * Get resource level auth providers as a list of strings.
```

File: components/micro-gateway-core/src/main/ballerina/src/gateway/utils/analytics_util.bal [hunk 3]
```diff
@@ -118,7 +118,6 @@ function populateFaultAnalyticsDTO(http:FilterContext context, string err) retur
     eventDto.apiCreatorTenantDomain = getTenantDomain(context);
     eventDto.hostName = retrieveHostname(DATACENTER_ID, <string>context.attributes[HOSTNAME_PROPERTY]);
     eventDto.protocol = <string>context.attributes[PROTOCOL_PROPERTY];
-
     runtime:InvocationContext invocationContext = runtime:getInvocationContext();
     if (isSecured && invocationContext.attributes.hasKey(AUTHENTICATION_CONTEXT)) {
```
Comment (praminda) [inline components/micro-gateway-core/src/main/ballerina/src/gateway/utils/analytics_util.bal:122]:
```suggestion

    if (isSecured && invocationContext.attributes.hasKey(AUTHENTICATION_CONTEXT)) {
```

File: tests/src/test/java/org/wso2/micro/gateway/tests/common/Constants.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (praminda) [inline tests/src/test/java/org/wso2/micro/gateway/tests/common/Constants.java:2]: shouldn't this be 2020?
```suggestion
 * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

**Review by Rajith90 — APPROVED** [review]

**Review by praminda — APPROVED** [review]

**Pull request merged**